property_caching/__init__.py

```diff
@@ -38,12 +45,14 @@ def _get_cache(obj, cache_attr_name=_OBJ_CACHE_ATTR_NAME):
     return getattr(obj, cache_attr_name, {})
 
 
-def _get_property_value(fn, obj, cache_attr_name):
+def _get_property_value(fn, obj, cache_attr_name, only_cache_true_vals=False):
+    result = None
```

Comment:
I need it for line 58:

```python
return cache.get(fn.__name__, result)
```

 I don't want to use None as the default there because it would swallow things like `[]`.